scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/StepSimulationTest.java

```diff
@@ -0,0 +1,64 @@
+package gov.nasa.jpl.aerie.scheduler;
+
+import gov.nasa.jpl.aerie.merlin.driver.SerializedActivity;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import org.apache.commons.lang3.tuple.Triple;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.SECOND;
+
+public class StepSimulationTest {
+
+  @Test
+  public void resetEveryTime() {
+    manyacts(true);
+  }
+  @Test
+  public void dontreset() {
+    manyacts(false);
+  }
```

Comment:
This class smells less like a test and more like a top-level program that happens to be a benchmark, especially because there are no assertions being checked. Since this is a benchmark, we don't really want ot ti be run alongside all the other unit tests; unit tests should be fast!

Recommend making this method a top-level `main`, and removing the other method (`resetEveryTime`), per earlier comment about removing the performance-test-only resetting toggle.

Reply:
![Performance comparison](https://user-images.githubusercontent.com/94930916/149028535-50966129-6039-41ec-a326-1ebfe87b032c.png)
Just for you @Twisol (this is on Slack), this is the graphical version of what is printed during the tests (not sure this is clear), showing the difference in performance between the incremental and reset versions. This is exciting, I hope it translates when we use it with scheduling.

Reply:
This is exactly what you are saying, a benchmark. I will make this a program. About the other method, I'll answer where you commented.

Reply:
Yes, the graph makes sense; you're not re-running the simulation every time you add an activity in the future. This is an exciting optimization; I'm really hopeful that this can be refined, over time, to replace the existing driver altogether.